charts/tempo-distributed/values.yaml

```diff
@@ -316,16 +342,42 @@ config: |
         store: memberlist
   distributor:
     receivers:
+      {{ if .Values.traces.jaeger.enabled }}
       jaeger:
         protocols:
-          grpc:
-            endpoint: 0.0.0.0:14250
-          thrift_binary:
-            endpoint: 0.0.0.0:6832
+          {{- if .Values.traces.jaeger.thriftCompact }}
           thrift_compact:
             endpoint: 0.0.0.0:6831
+          {{- end }}
+          {{- if .Values.traces.jaeger.thriftBinary }}
+          thrift_binary:
+            endpoint: 0.0.0.0:6832
+          {{- end }}
+          {{- if .Values.traces.jaeger.thriftHttp }}
           thrift_http:
             endpoint: 0.0.0.0:14268
+          {{- end }}
+          {{- if .Values.traces.jaeger.grpc }}
+          grpc:
+            endpoint: 0.0.0.0:14250
+          {{- end }}
+      {{- end }}
+      {{- if .Values.traces.zipkin}}
+      zipkin:
+        endpoint: 0.0.0.0:9411
+      {{- end }}
+      {{- if .Values.traces.otlp}}
+      otlp:
+        protocols:
+          http:
+            endpoint: 0.0.0.0:55680
```

Comment:
does this work?  opening both to 55680?  i feel like this would cause a port conflict.

Reply:
Split the options for otlp. so only 1 endpoint will be enabled.